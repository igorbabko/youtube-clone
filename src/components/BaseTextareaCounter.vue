<template>
  <div class="absolute bottom-2 right-2 text-xs text-gray-600 peer-focus:text-black">
    <slot />
  </div>
</template>
