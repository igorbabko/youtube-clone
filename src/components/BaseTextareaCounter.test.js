import { render } from '@testing-library/vue'
import BaseTextareaCounter from './BaseTextareaCounter.vue'

it('renders correctly', () => {
  const options = {
    slots: {
      default: '0 / 100'
    }
  }

  const { html } = render(BaseTextareaCounter, options)

  expect(html()).toMatchSnapshot()
})
