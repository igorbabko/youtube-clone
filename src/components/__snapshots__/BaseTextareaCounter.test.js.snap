// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"<div class=\\"absolute bottom-2 right-2 text-xs text-gray-600 peer-focus:text-black\\">0 / 100</div>"`;
